src/status/SwapExpired.tsx

```diff
@@ -40,6 +43,10 @@ const SwapExpired = () => {
                     swap={swap as Accessor<SubmarineSwap | ChainSwap>}
                 />
                 <hr />
+                <BlockExplorer
+                    asset={swap().assetSend}
+                    txId={transactionToRefund()}
```

Comment:
This cannot work because you are setting `setTransactionToRefund(res.hex);`; the raw hex of the transaction, not the ID